brayns/common/PropertyMap.h

```diff
@@ -273,12 +288,6 @@ inline PropertyMap::Property::Type
     return PropertyMap::Property::Type::String;
 }
 template <>
-inline PropertyMap::Property::Type
```

Comment:
It was causing issues since a String could be either a const char* or an std::string, when you do not know how it was created you get exceptions if you use the incorrect type. I prefer having a 1:1 mapping between the types and their storage.